Compilation fails with stropts.h: No such file or directory

```
$ make
..
fakeboard.c:10:10: fatal error: stropts.h: No such file or directory
   10 | #include <stropts.h>
      |          ^~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:23: fakeboard.o] Error 1
```

--- a/src/fakeboard.c
+++ b/src/fakeboard.c
@@ -7,7 +7,6 @@
 #include <errno.h>
 #include <unistd.h>
 #include <fcntl.h>
-#include <stropts.h>
 #include <sys/ioctl.h>
 
 #include <linux/input.h>

